native/index.d.ts

```diff
@@ -0,0 +1,68 @@
+// <reference path="../react/react-native.d.ts" />
```

Comment:
In general declaration files should create dependencies between each other through `import`s, not `reference`s.